db/migrate/20181020103501_revoke_variant_overrideswithout_permissions.rb

```diff
@@ -0,0 +1,17 @@
+class RevokeVariantOverrideswithoutPermissions < ActiveRecord::Migration
+  def up
+    # This process was executed when the permission_revoked_at colum was created (see AddPermissionRevokedAtToVariantOverrides)
+    #   It needs to be repeated due to #2739
+    variant_override_hubs = Enterprise.where(id: VariantOverride.all.map(&:hub_id).uniq)
```

Comment:
You probably just took it from the previous migration, but this call is a bit clearer and more efficient:
```suggestion
    variant_override_hubs = Enterprise.where(id: VariantOverride.uniq.pluck(:hub_id))
```

It also enables ActiveRecord to nest the VariantOverride query within the Enterprise query instead of making two queries.

Reply:
yeah, this was copy pasted from the original revoked_at code. I tested it with an example.

It would probably be better writing SQL, correct? Things like this will break in the future if these things are removed from the model : 
"hub.relationships_as_child.with_permission"

Hmm. what do I do? Is there an easy fix? Can I ignore this for now? We will have to look at it when/if it breaks in the future.

Reply:
thanks for the pluck lesson!

Reply:
Sorry to be late but I totally missed this PR. My concern is that `VariantOverride.all` can potentially blow up some instance's DB. That turns into a `SELECT` without `WHERE` constraints thus, selecting the whole table :boom: I suggest we process them in chunks.

Reply:
ok, do you think this query can blow a database? "select id from variant_overrides;"
Canada has 908 entries.

Anyway, I can re-write, can you please guide me here. What's the proper way of doing this?